test/unit/reductions/grid_search/test_grid_search_arguments.py

```diff
@@ -21,29 +27,40 @@
 candidate_Y_transforms = conversions_for_1d
 candidate_A_transforms = conversions_for_1d
 
+def _skip_if_invalid_transformation(A, transform):
+    if len(A.shape) > 1 and A.shape[1] > 1 and transform in [ensure_list, ensure_series]:
+        pytest.skip()
```

Comment:
This is executable code in the middle of a test? Does this mean we end up with lots of skipped tests in our results?

Reply:
Yes we do skip some, but they're marked as such. This is from running unit tests locally
```
 5014 passed, 526 skipped in 398.52 seconds
```

Reply:
I was afraid of that. I think it's strange to flag a scenario as skipped when it's never intended to run.

Reply:
I don't think it's problematic, especially if the reason for skipping is mentioned. I agree that it's nicer if they never get generated in the first place but I doubt it's worth holding off on this right now. There are many more things we should do to make our tests nicer and maybe built on the same foundation, and that will hopefully address this issue as well.

Reply:
fixing this in the other PR. Essentially they'll be "deselected" instead of skipped. I hope that's sufficient. Closing this as duplicate of the other thread.